qa/rpc-tests/bipdersig-p2p.py

```diff
@@ -102,7 +102,7 @@ def get_tests(self):
             self.last_block_time += 1
             self.tip = block.sha256
             height += 1
-        yield TestInstance(test_blocks, sync_every_block=False)
+        yield TestInstance(test_blocks, sync_every_block=True)
```

Comment:
agree , it does slow it down but is necessary because of PV...you might want to also consider on line 92 adding sync_every_block=True  if there are more problems.